researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/BridgeDataProvider.java

```diff
@@ -232,13 +235,28 @@ protected ConsentSignature createConsentSignatureBody(TaskResult consentResult)
 
         // Save Consent Information
         // User is not signed in yet, so we need to save consent info to disk for later upload
-        return new ConsentSignature().name(fullName).birthdate(new LocalDate(birthdateInMillis)).imageData(base64Image).imageMimeType("image/png").scope(SharingScope.valueOf(sharingScope));
+        return new ConsentSignature()
+                .name(fullName)
+                .birthdate(new LocalDate(birthdateInMillis))
+                .imageData(base64Image)
+                .imageMimeType("image/png")
```

Comment:
This is what ConsentSignatureStepLayout returns and there's no other use case that I know of, so I think it can be deferred. I'll create a JIRA